src/containers/CharacterProfile/index.js

```diff
@@ -38,11 +38,16 @@ class CharacterProfileScreen extends Component {
     this.props.dispatch(fetchDataForCharacter(this.props.characterID));
   }
 
+  filteredAttacks(moves) {
```

Comment:
Ideally, I would move this to an action in character where we would filter out the moves. Then on render, we check if there are filtered moves to show, if not, show all moves.

Keep our data logic outside of our display logic.